lab-examples/clos02/connectivity.csv

```diff
@@ -0,0 +1,57 @@
+Source,Interface,Towards,IPv4,IPv4 Prefix,IPv6,IPv6 Prefix
```

Comment:
what is this csv for?

Reply:
It denotes the complete IP assignment information for clos02 topology.

Reply:
Will be added into README itself.

Reply:
yes, I think this can be removed from the repo and instead the table in the readme should be used instead

Reply:
Have already added.